elephant/conversion.py

```diff
@@ -152,3 +155,704 @@ def binarize(spiketrain, sampling_rate=None, t_start=None, t_stop=None,
+            Returns a dense matrix representation of the sparse matrix,
+            with ones indicating a spike and zeros for non spike.
+            The ones in the columns represent the index
+            position of the spike in the spike train and rows represent the
+            number of spike trains.
+
+        Examples
+        --------
+        >>> import elephant.conversion as conv
+        >>> import neo as n
+        >>> import quantities as pq
+        >>> a = n.SpikeTrain([0.5, 0.7, 1.2, 3.1, 4.3, 5.5, 6.7] * pq.s, t_stop=10.0 * pq.s)
+        >>> x = conv.Binned(a, num_bins=10, binsize=1 * pq.s, t_start=0 * pq.s)
+        >>> print(x.mat_clipped)
+        [[1 1 0 1 1 1 1 0 0 0]]
+
+        See also
+        --------
+        scipy.sparse.csr_matrix
+        scipy.sparse.csr_matrix.toarray
+        """
+        return abs(scipy.sign(self.mat_unclipped))
+
+    @property
+    def mat_unclipped(self):
+        """
+        Returns the sparse matrix, which rows represents the number of
+        spike trains and the columns represents the binned index position
+        of a spike in a spike train.
+        The  matrix columns contain the number of spikes that
+        occurred in the spike train(s).
+        If **bool** :attr:`store_mat` is set to **True** last calculated
+        `unclipped` matrix will be returned.
+
+        Returns
+        -------
+        unclipped matrix : numpy.ndarray
+            Matrix with spike times. Columns represent the index position of
+            the binned spike and rows represent the number of spike trains.
+
+        Examples
+        --------
+        >>> import elephant.conversion as conv
+        >>> import neo as n
+        >>> a = n.SpikeTrain([0.5, 0.7, 1.2, 3.1, 4.3, 5.5, 6.7] * pq.s, t_stop=10.0 * pq.s)
+        >>> x = conv.Binned(a, num_bins=10, binsize=1 * pq.s, t_start=0 * pq.s)
+        >>> print(x.mat_unclipped)
+        [[2 1 0 1 1 1 1 0 0 0]]
+
+        See also
+        --------
+        scipy.sparse.csr_matrix
+        scipy.sparse.csr_matrix.toarray
+
+        """
+        if self.mat_u is not None:
+            return self.mat_u
+        if self.store_mat_u:
+            self.mat_u = self.sparse_mat_unclip.toarray()
+            return self.mat_u
+        # Matrix on demand
+        else:
+            return self._sparse_mat_u.toarray()
+
+    def store_mat_unclipped(self):
+        if self.mat_u is None:
+            self.mat_u = self.sparse_mat_unclip.toarray()
+            self.store_mat_u = True
+
+    def __convert_to_binned(self, spiketrains):
+        """
+        Converts neo.core.SpikeTrain objects to a sparse matrix
+        (`scipy.sparse.csr_matrix`), which contains the binned times.
+
+        Parameters
+        ----------
+        spiketrains : neo.SpikeTrain object or list of SpikeTrain objects
+           The binned time array :attr:`spike_indices` is calculated from a
+           SpikeTrain object or from a list of SpikeTrain objects.
+
+        Examples
```

Comment:
This example does not invoke `__convert_to_binned`.  You really don't need an example for non-public API.